<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="Liku" name="Test" tests="2" failures="2" timestamp="11 Jun 2019 16:03:34 GMT" time="16.727" errors="0">
  <testcase name="TC01_StudentRegistration" time="7.03" classname="testCases.StudentRegistration">
    <failure type="java.lang.AssertionError" message="Student Registration Failed due to exception!">
      <![CDATA[java.lang.AssertionError: Student Registration Failed due to exception!
	at org.junit.Assert.fail(Assert.java:88)
	at pageObjectsRepository.CETSignUpPage.studentRegistration(CETSignUpPage.java:68)
	at testCases.StudentRegistration.TC01_StudentRegistration(StudentRegistration.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TC01_StudentRegistration -->
  <testcase name="TC01_StudentRegistration" time="7.03" classname="testCases.StudentRegistration">
    <failure type="java.lang.AssertionError" message="Student Registration Failed due to exception!">
      <![CDATA[java.lang.AssertionError: Student Registration Failed due to exception!
	at org.junit.Assert.fail(Assert.java:88)
	at pageObjectsRepository.CETSignUpPage.studentRegistration(CETSignUpPage.java:68)
	at testCases.StudentRegistration.TC01_StudentRegistration(StudentRegistration.java:45)
... Removed 25 stack frames]]>
    </failure>
  </testcase> <!-- TC01_StudentRegistration -->
</testsuite> <!-- Test -->
